Client/WebFilters/ShieldStats/Adblock/Resources/ablock_rust_lib.h

```diff
@@ -0,0 +1,96 @@
+#ifndef ADBLOCK_RUST_FFI_H
```

Comment:
This file was generated, see https://github.com/brave/adblock-rust-ffi#regenerating-the-c-header